### PR TITLE
adding first file

### PR DESCRIPTION
**Pull request by Kakalanp**
In this milestone, we added unit testing using RSpec for the following classes:

- Book

- Person

- Student

- Teacher

- Classroom

- Rental

- Nemeable

- BaseDecorator

- CapitalizeDecorator

- TrimmerDecorator

### CODE REVIEW

File: spec/teacher_class_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require './teacher_class'
+
+describe Teacher do
```
Comment (SajjadAhmad14) [inline spec/teacher_class_spec.rb:3]: Please add unit tests for all public methods of `Teacher`, `Student`, and `Book` classes.

File: spec/classroom_class_spec.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require './classroom_class'
+
+describe 'Classroom' do
```
Comment (SajjadAhmad14) [inline spec/classroom_class_spec.rb]: Please add unit test for `#add_student` in `Classroom` class.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi team, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏
- [x] Please add a command `rspec` in README so a user is able to run the tests.
- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by richardoppiyo** [comment]
The changes have been made in the above commit as requested by the reviewer

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


@Kakalanp @richardoppiyo  ✋🏼 
#### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
___
### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
- [x] Like the last code reviewer said please add this in your readme file `To test the project, run rspec in your terminal` so a user is able to run the tests.

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by richardoppiyo** [comment]
The README file has been updated as required

**Review by Lameck1 — APPROVED** [review]
Hi, @Kakalanp & @richardoppiyo,

You've done well addressing changes from previous reviews 👏🏼.
Tasks for this project are now successfully implemented 💯.

### Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by richardoppiyo** [comment]
Thank you

**Pull request merged**